torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1276,6 +1282,12 @@ def unpack_variables(args):
 EXCLUDE_GRADCHECK = {
 }
 EXCLUDE_GRADGRADCHECK = {
+    'fft.fft',
```

Comment:
No double backward support is OK - what's the blocker on double backward support, though?

Reply:
I believe it's `_fft_with_size` that's failing `gradgradcheck`. I assume it just wasn't written with gradgrad in mind. I can revisit this later; I don't think there's a fundamental reason it couldn't work.